enterprise/app/history/history_invocation_card.tsx

```diff
@@ -214,7 +217,7 @@ export default class HistoryInvocationCardComponent extends React.Component {
                   {this.props.invocation.host}
                 </div>
               )}
-              {this.props.invocation.command && (
+              {this.props.invocation.command && this.props.invocation.command != "workflow run" && (
```

Comment:
can we keep the tool icon?

Reply:
woops, done. This made more sense when rendering the role as one of the details (it looked awkward having 2 details that both said "workflow")